lib/EasyPost/CarrierAccount.php

```diff
@@ -76,7 +77,14 @@ public static function create($params = null, $apiKey = null)
             unset($params);
             $params['carrier_account'] = $clone;
         }
-        return self::createResource(get_class(), $params, $apiKey);
+        $type = $params['carrier_account']['type'];
+        if (is_null($type)) {
```

Comment:
For PHP, you'll want this instead. Otherwise we won't get the intended affect:

```suggestion
        if (!isset($type)) {
```